EventsExpress/ClientApp/src/components/shared/errorMessage.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+
+
+export default class ErrorMessages extends Component {
+
+    render() {
+        return (
+            <>
+                {this.props.error.map(x => <div className="text-danger text-center">{x}</div>)}
```

Comment:
I think we should set align text mode to 'center' here but get it from props instead, so the parent component will have the opportunity to manage it.